docs/tutorials/synchronisation.rst

```diff
@@ -165,7 +165,7 @@ ETags are provided in response headers, for the collection as well as individual
 records.
 
 Even though it is recommended to consider them as opaque and abstract, it can still
-be useful to notice that ETags are a string with the quoted record last modified value
+be useful to notice that ETags are the ``last_modified`` value of the record, quoted
```

Comment:
I think this is fine, but having `quoted` at the end of the noun phrase feels a little clumsy to me. Normally I'd probably write `ETags are the quoted ``last_modified`` value of the record`, but I see that the example `(``"<record.last_modified>"``)` being next to the word "quoted" is kind of nice too.

I'd probably also use "observe" rather than "notice" here, but I think notice is OK.